### PR TITLE
parlay: init at 0.6.0

### LINKED ISSUE #324547: Package request: parlay
**Project description**

**Parlay** is an open source tool from Snyk that enriches SBOMs
The increasing adoption of software bill of materials (SBOM) standards are starting to drive better interoperability between security tools.

**Metadata**

* homepage URL: https://snyk.io/blog/introducing-parlay/
* source URL: https://github.com/snyk/parlay
* license:  Apache-2.0 license
* platforms: linux, darwin, ...

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by kiike**
## Things done

Created a package for parlay. Closes #324547. cc @kefniark (let me know if you're interested in maintaining this).

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by kiike** [comment]
To test, it's enough to do

```bash
curl https://raw.githubusercontent.com/snyk/parlay/main/testing/sbom.cyclonedx.json | ./result/bin/parlay ecosystems enrich -
```

**Comment by kefniark** [comment]
> @kefniark (let me know if you're interested in maintaining this).

Yes sure, I'm using the tool on regular basis so I can keep updating it.
For example, there was a version `v0.5.1` few days ago https://github.com/snyk/parlay/releases

Is there anything to do to get this PR reviewed and merged?

**Comment by kiike** [comment]
> > @kefniark (let me know if you're interested in maintaining this).
> 
> Yes sure, I'm using the tool on regular basis so I can keep updating it. For example, there was a version `v0.5.1` few days ago https://github.com/snyk/parlay/releases

I added you to the maintainer list.
 
> Is there anything we can do to get this PR reviewed and merged? I feel that when the PR drift over page 50+, it's unlikely to get any attention anymore 😄

You're right, it will probably slip under all radars, just like it did for me for over a month (sorry for the late reply :pray:).

Maybe it will be helpful if you can follow the checklist in https://discourse.nixos.org/t/prs-already-reviewed/2617 and then link the PR there.

File: pkgs/by-name/pa/parlay/package.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    rev = "v${version}";
+    hash = "sha256-9drACfJ8iA6qym7BQywJxzchQcFch4IW8/6q5dWb3KA=";
+  };
+
+  vendorHash = "sha256-Eo5MgdISiwbaJFg5XHAwe5x3D8GmgzswYmcUG4gvaQk=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Enriches SBOMs with data from third party services ";
```
Comment (FliegendeWurst) [inline pkgs/by-name/pa/parlay/package.nix:24]:
```suggestion
    description = "Enriches SBOMs with data from third party services";
```

**Review by FliegendeWurst — APPROVED** [review]

File: pkgs/by-name/pa/parlay/package.nix (hunk 1 above)
Comment (kefniark) [reply pkgs/by-name/pa/parlay/package.nix:24]: @kiike can you approve this change please?

**Review by kefniark — APPROVED** [review]

File: pkgs/by-name/pa/parlay/package.nix (hunk 1 above)
Comment (kiike) [reply pkgs/by-name/pa/parlay/package.nix]: Hi! Thanks for the suggestion! I've committed the change and squashed it. You're attributed as co-author.

**Comment by kiike** [comment]
Took the chance to update the package too. The release happened during the  NixCon and didn't notice it :sweat_smile:

**Review by FliegendeWurst — APPROVED** [review]

**Pull request merged**